### PR TITLE
Bugfix/fix encoding

### PR DESCRIPTION
**Pull request by gbrlcustodio**
Improves encoding support by using browser native's textdecoder (also included as a polyfill). An use case example can be found at the very last test spec.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,5 +41,8 @@
     "eslint-plugin-prettier": "^3.1.1",
     "jest": "^24.9.0",
     "prettier": "^1.18.2"
+  },
```
Comment (brunoocasali) [inline package.json]: You also, could provide an minor version increase in package.json ;)
Or do you plan change it in another PR?

Comment (gbrlcustodio) [reply package.json]: My bad, actually. Bumping it right now
